### ISSUE TITLE
Uploads via File App is always stuck

### ISSUE THREAD

**Issue by gitlik** [nextcloud/ios#2950]
It worked for years, it doesn't anymore. I have two folders, nested inside another one, they are called

→ Steuer
  → Freiberuf
  → Gewerbe

My scanner app used to automatically uploaded new scans there directly, but no matter how I put files in there, the upload gets stuck. 

First, I've been thinking it might be the special character arrow in the beginning, but no matter how I change the name it doesn't help. 

![image](https://github.com/nextcloud/ios/assets/26041973/14a3d5ba-a8b9-416c-b04f-26fcc05847a9)
![image](https://github.com/nextcloud/ios/assets/26041973/f70aba65-0c36-41b6-98fb-5f26f7a0ed7f)

**Comment by antonisar** [comment]
No. The bug appears to be that it takes the “upload folder” setting from auto upload wrong 

e.g. Photos\2024\06\filexxx is added to the queue as multiple uploads  Photos\2024\06 Photos\2024\06\filexxx, Photos\2024\06\2024\06\filexxx,  Photos\2024\06\2024\06\2024\06\2024\06\filexxx and of course those folders do not exist
